client/src/'

```diff
@@ -0,0 +1,25 @@
+import { createGlobalStyle } from 'styled-components';
```

Comment:
What is the purpose of this file? It seems like a copy of globalStyles.js, and I can't see any place where it is used.

Reply:
Well that was a mistake, thanks for catching that, will remove